libs/checkpoint/package.json

```
@@ -6,21 +6,20 @@
   "engines": {
     "node": ">=18"
   },
-  "main": "./index.js",
-  "types": "./index.d.ts",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "repository": {
     "type": "git",
     "url": "git@github.com:langchain-ai/langgraphjs.git"
   },
   "scripts": {
     "build": "yarn turbo:command build:internal --filter=@langchain/langgraph-checkpoint",
-    "build:internal": "yarn clean && yarn lc_build --create-entrypoints --pre --tree-shaking",
+    "build:internal": "yarn workspace @langchain/build compile @langchain/langgraph-checkpoint",
```

I think ideally we we don't need to specify package name here

---

You can call `yarn workspace @langchain/build compile` to compile all packages but in order to just compile a specific workspace you have to provide a filter. Alternatively I think:
```suggestion
    "build:internal": "yarn workspace @langchain/build compile $npm_package_name",
```

should work?